protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -0,0 +1,110 @@
+package protokube
+
+import (
+	"fmt"
+	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/util/intstr"
+	"strings"
+)
+
+func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
```

Comment:
Adding one to describe the public function.  Otherwise I don't know where a comment would add value.